playbooks/roles/shadowsocks/templates/instructions.md.j2

```diff
@@ -64,6 +64,17 @@ For reference, this is the configuration specified:
         Password: {{ shadowsocks_password.stdout }}
         Encryption Method: {{ shadowsocks_encryption_method }}
 
+For users behind stricter surveillance (esp. experiencing QOS throttling in PRC), further configuration of shadowsocks along with simple-obfs plugin can be carried out via following configuration:
+
+        Server: {{ streisand_ipv4_address }}
+        Port: {{ shadowsocks_server_port }}
+        Password: {{ shadowsocks_password.stdout }}
+        Encryption Method: {{ shadowsocks_encryption_method }}
+        Plugin: {{ shadowsocks_obfs_plugin }}
+        Plugin_Options: {{ shadowsocks_obfs_client_plugin_opts }}
+
+Note: www.xxx.com should be replaced by a known host name like, www.cloudfront.com or www.baidu.com etc.
```

Comment:
No it isn't replaced server side. It is only mentioned for the client side. Even without setting it, shadowsocks traffic works fine in my tests (client side). However, I have access to Linux based SS-libev client, I checked windows version but it doesn't support this option as yet. That is why I only edited the Linux section. Android client also has an option of disabling/enabling `simple-obfs` as per requirements.

So this setting is **optional** for client side only. Server side `simple-obfs` can be enabled by default and it doesn't affect the non-obfs traffic from clients.